packages/twenty-ui/src/navigation/menu-item/internals/components/StyledMenuItemBase.tsx

```diff
@@ -35,10 +36,16 @@ export const StyledMenuItemBase = styled.div<MenuItemBaseProps>`
   ${({ theme, isKeySelected }) =>
     isKeySelected ? `background: ${theme.background.transparent.light};` : ''}
 
-  ${({ isHoverBackgroundDisabled }) =>
-    isHoverBackgroundDisabled ?? HOVER_BACKGROUND};
+  ${({ isHoverBackgroundDisabled, disabled }) =>
+    (disabled || isHoverBackgroundDisabled) ?? HOVER_BACKGROUND};
+
+  ${({ theme, accent, disabled }) => {
+    if (disabled !== undefined && disabled !== false) {
```

Comment:
cannot be undefined because of the default

Reply:
if (disabled)

Reply:
Actually this one can be undefined, because we're in another component that doesn't require `disabled` in the props and doesn't init it by default as it's a styled component.

Reply:
OK, in this case please us isUndefinedGuard